xds/internal/xdsclient/xdsresource/unmarshal_cds.go

```diff
@@ -463,3 +483,105 @@ func circuitBreakersFromCluster(cluster *v3clusterpb.Cluster) *uint32 {
 	}
 	return nil
 }
+
+// outlierConfigFromCluster extracts the relevant outlier detection configuration from
+// the received cluster resource. Returns nil if no OutlierDetection.
```

Comment:
The comments ends abruptly. Could you please complete the comment.

Reply:
I don't get what you mean by "ends abruptly". I patterned this after the other xFromCluster methods in unmarshal_cds. Anyways, I switched this comment to "outlierConfigFromCluster extracts the relevant outlier detection configuration from the received cluster resource. Returns nil if no OutlierDetection field set in the cluster resource."